site/content/docs/user/configuration.md

```diff
@@ -304,7 +304,7 @@ Extra port mappings can be used to port forward to the kind nodes. This is a
 cross-platform option to get traffic into your kind cluster. 
 
 If you are running Docker without the Docker Desktop Application on Linux, you can simply send traffic to the node IPs from the host without extra port mappings. 
-With the installation of the Docker Desktop Application, whether it is on macOs, Windows or Linux, you'll want to use these.
+Without the installation of the Docker Desktop Application, whether it is on macOs, Windows or Linux, you'll want to use these.
```

Comment:
I believe the original text was accurate, though this whole section could probably be reworded to make it more clear. Basically it is saying:

- If you are using Docker without Docker Desktop, you don't need extra port mappings.
- If you are using Docker Desktop you do need extra port mappings.

I believe this also now applies to more than just Docker Desktop: colima, Rancher Desktop, etc. Basically anything that enables running Docker on the OS by putting it inside a VM.

Reply:
Ok, thanks for the clarification. 

I don't if you're familiar with the [AWS Cloud9](https://aws.amazon.com/cloud9/) service. I am running one. I had to configure port mapping to reach my pod exposed via a NodePort service. Does it mean that Docker Desktop is installed on my Linux machine which has no GUI?

Reply:
Hmm, good question @network-charles. I'm not actually sure what they are using in Cloud9. I would expect either Docker directly, or maybe Finch. It's possible they are doing some sort of special network handling that is causing problems.

Reply:
@stmcginnis, actually I meant that I had to use that configuration to reach my pod from my non-remote machine. Sorry, I wasn't clear enough.

curl traffic `non-remote machine > remote machine > kind node > k8s service > k8s pod`

I just deployed a regular virtual machine on AWS without using Cloud9 and I experienced the same issue. I had to configure port mapping to reach the pod from my non-remote machine.

Reply:
The initial documentation could still be correct. Maybe we need to add a new sentence about accessing the Kind node if you are running Kind on a remote machine and need to send traffic to the Kind node's IP from a non-remote machine.